functions-and-scope.js

```diff
@@ -82,3 +128,17 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // highestGrade(grades) geeft 9
 // highestGrade([6, 4, 5]) geeft 6
 // highestGrade([8, 9, 4, 6, 10]) geeft 10
+
+function highestGrades(grades) {
+    let highestGrade = 0;
```

Comment:
Het is beter om te beginnen met de eerste waarde van de array in plaats van `0`. Nu gaat het in dit voorbeeld misschien niet helemaal op wat ik nu ga uitleggen: Stel je voor dat je een array heeft van [-2, -4, -1, -5]. Als je deze nu in deze functie stopt, dan komt er `0` uit, terwijl deze niet eens in de array voorkomt.

```suggestion
    let highestGrade = grades[0];
```